spec/unit/matrix-client.spec.ts

```diff
@@ -1394,36 +1397,40 @@ describe("MatrixClient", function () {
                 );
             });
 
-            describe("and the server supports relation based redactions (unstable)", () => {
-                beforeEach(async () => {
-                    unstableFeatures["org.matrix.msc3912"] = true;
-                    // load supported features
-                    await client.getVersions();
-                });
-
-                it("should send with_relations in the request body", async () => {
-                    const txnId = client.makeTxnId();
+            // expects both string[] and string to work
+            describe.each([[RelationType.Reference], RelationType.Reference])(
+                "and the server supports relation based redactions (%s, unstable)",
+                (relationType: RelationType | Array<RelationType>) => {
+                    beforeEach(async () => {
+                        unstableFeatures["org.matrix.msc3912"] = true;
+                        // load supported features
+                        await client.getVersions();
+                    });
 
-                    httpLookups = [
-                        {
-                            method: "PUT",
-                            path:
-                                `/rooms/${encodeURIComponent(roomId)}/redact/${encodeURIComponent(eventId)}` +
-                                `/${encodeURIComponent(txnId)}`,
-                            expectBody: {
-                                reason: "redaction test",
-                                [MSC3912_RELATION_BASED_REDACTIONS_PROP.unstable!]: [RelationType.Reference],
+                    it("should send with_rel_types in the request body", async () => {
+                        const txnId = client.makeTxnId();
+
+                        httpLookups = [
+                            {
+                                method: "PUT",
+                                path:
+                                    `/rooms/${encodeURIComponent(roomId)}/redact/${encodeURIComponent(eventId)}` +
+                                    `/${encodeURIComponent(txnId)}`,
+                                expectBody: {
+                                    reason: "redaction test",
+                                    [MSC3912_RELATION_BASED_REDACTIONS_PROP.unstable!]: [RelationType.Reference],
```

Comment:
to ensure that we aren't making the same mistake twice (and to make the test clearer), I think it is better to hardcode the property names here:

```suggestion
                                    "org.matrix.msc3912.with_relations": ["m.reference"],
```

Likewise the reference to `RelationType` above.